### ISSUE TITLE
if the fired url matches the current one, should the event be prevented?

### ISSUE THREAD

**Issue by fponticelli** [joelpellucid/routly#4]

**Comment by joelpellucid** [comment]
Yep, we should not fire the event if the route dispatched matches the current window.location.hash.

Routly still does not remember what the current hash is.  This logic is taken care of by HtmlRouteEmitter.hx.  See commit 23462996df1f2b31e4e0e9608355cea2ad7d8bf4.
